tools/cmake/toolchains/ARM.cmake

```diff
@@ -64,6 +64,13 @@ function(mbed_set_toolchain_options target)
         )
     endif()
 
+    # Add linker configuration to generate a map file
```

Comment:
Similar to the change made in `GCC_ARM.cmake`, this comment should be moved right above appending the linker option or be rephrased `Add linker option to generate a map file if required.`